themes/default/register.tmpl.php

```diff
@@ -73,13 +73,13 @@ function encrypt_password()
 
 		<tr>
 			<td align="left"><div class="required" title="<?php echo _AC('required_field'); ?>">*</div><label for="login"><?php echo _AC('login_name'); ?></label>:</td>
-			<td align="left"><input id="login" name="login" type="text" maxlength="20" size="30" value="<?php if (isset($_POST['login'])) echo $stripslashes(htmlspecialchars($_POST['login'])); else echo $stripslashes(htmlspecialchars($this->user_row['login'])); ?>" /></td>
```

Comment:
The `maxlength` still need to be specified to prevent input lengths go beyond what's defined for the corresponding table field.

Reply:
Thanks for Reviewing.